From 64a1ba6047bd9671a68c6e168cc171cb28cc2b44 Mon Sep 17 00:00:00 2001
From: Andrew Pinski <apinski@marvell.com>
Date: Mon, 15 Jul 2019 12:11:55 -0700
Subject: [PATCH 343/386] arm64: Add MIDR encoding for Marvell OcteonTX 2
 Fussion

This patch adds the MIDR encodings for Marvell's
OcteonTX2 Fussion.

Change-Id: I79546cc1b1e38591b86249107e19d2af6b7057c0
Signed-off-by: Andrew Pinski <apinski@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/12597
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/include/asm/cputype.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/include/asm/cputype.h b/arch/arm64/include/asm/cputype.h
index 6e7caa788976..2e844fe12a71 100644
--- a/arch/arm64/include/asm/cputype.h
+++ b/arch/arm64/include/asm/cputype.h
@@ -95,6 +95,7 @@
 #define CAVIUM_CPU_PART_THUNDERX2	0x0AF
 
 #define MRVL_CPU_PART_OCTEONTX2_96XX	0x0B2
+#define MRVL_CPU_PART_OCTEONTX2_95XX	0x0B3
 
 #define BRCM_CPU_PART_VULCAN		0x516
 
@@ -117,6 +118,7 @@
 #define MIDR_OCTEON_T83 MIDR_CPU_MODEL(ARM_CPU_IMP_CAVIUM, CAVIUM_CPU_PART_T83)
 #define MIDR_CAVIUM_THUNDERX2 MIDR_CPU_MODEL(ARM_CPU_IMP_CAVIUM, CAVIUM_CPU_PART_THUNDERX2)
 #define MIDR_MRVL_OCTEONTX2_96XX MIDR_CPU_MODEL(ARM_CPU_IMP_CAVIUM, MRVL_CPU_PART_OCTEONTX2_96XX)
+#define MIDR_MRVL_OCTEONTX2_95XX MIDR_CPU_MODEL(ARM_CPU_IMP_CAVIUM, MRVL_CPU_PART_OCTEONTX2_95XX)
 #define MIDR_BRCM_VULCAN MIDR_CPU_MODEL(ARM_CPU_IMP_BRCM, BRCM_CPU_PART_VULCAN)
 #define MIDR_QCOM_FALKOR_V1 MIDR_CPU_MODEL(ARM_CPU_IMP_QCOM, QCOM_CPU_PART_FALKOR_V1)
 #define MIDR_QCOM_FALKOR MIDR_CPU_MODEL(ARM_CPU_IMP_QCOM, QCOM_CPU_PART_FALKOR)
-- 
2.17.1

